src/main/java/uk/gov/hmcts/reform/sandl/snlrules/controllers/DataExportController.java

```diff
@@ -28,7 +28,7 @@
     @Autowired
     private DroolsServiceFactory droolsServiceFactory;
 
-    @RequestMapping(value = "/exporthtml", params = "rulesDefinition")
+    @RequestMapping(value = "/exporthtml")
```

Comment:
I do not remember but for some reason it needs to be here, if it;s not there then it becomes mandatory parm or something

Reply:
It's exactly opposite, when it's here it causes problems, because route cannot be mapped.